tests/unit/components/TimeSeries.spec.ts

```diff
@@ -53,6 +53,60 @@ vi.mock("highcharts", async (importOriginal) => {
 });
 
 describe("time series", () => {
+  it("should initialise the chart with the correct options", async () => {
```

Comment:
Was this just a missing test? Didn't think the time series behaviour had changed, other than removing intervention labels.

Reply:
Yeah just backfilling